logstash-core/src/main/java/org/logstash/ackedqueue/QueueExceptionMessages.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+package org.logstash.ackedqueue;
+
+/**
+ * A public class holds number of descriptive messages are used during the interaction with acked queue.
+ */
+public class QueueExceptionMessages {
```

Comment:
Nit: I think these can safely belong with `QueueRuntimeException.java`.

Out-of-scope future-refactor note: it would be nice if `QueueRuntimeException` were abstract, and had a concrete implementation for each exception type that included its own message. This would encourage the throwing of a `QueueRuntimeException` toward type-safety and allow code that needs to handle `QueueRuntimeException` to safely handle specific things.